scripts/installer-win.bat

```diff
@@ -54,4 +56,6 @@ IF NOT EXIST installer\%OUTWX% powershell -Command Invoke-WebRequest -Uri %URLWX
 
 echo  #   installing jinja2   #
 pip install jinja2
+echo  #   installing piyaml   #
+pip install pyaml
```

Comment:
The module name is PyYAML, pyaml is something different, but this works because pyaml depends on PyYAML and it's automaticallty installed